.vsts-dotnet-ci.yml

```diff
@@ -13,20 +13,20 @@ resources:
 jobs:
 - template: /build/ci/job-template.yml
   parameters:
-    name: Centos_x64_NetCoreApp30
+    name: Centos_x64_NetCoreApp31
```

Comment:
I don't think we will support multiple versions of .netcoreapp 3.x. 
Can we simply make the whole thing NetCoreApp3?

Reply:
That makes sense, though I do think that specifying which version of NetCoreApp3 (3.0 or 3.1) we're using would be helpful if in the future we see bugs that are specific to 3.0 or 3.1, as you pointed out with respect to #4933 .

I will update these references to version 3.0/3.1 to only have version "3", if you do believe that specifying the exact iteration of NetCoreApp3 won't be helpful as I mentioned above.